webapp/src/components/llmbot_post/styles.tsx

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) 2023-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import styled from 'styled-components';
+
+export const PostBody = styled.div`
+`;
+
+export const ControlsBar = styled.div`
```

Comment:
I find it easier to put styled components in the files that own them. That's part of why styled components are nice because everything is together. Unless these are truly shared (I don't think they are) 

If there are a bunch I usually put them after the component so it looks nicer.